site/content/en/docs/api_sdk/sdk/lowlevel-api.md

```diff
@@ -303,20 +314,29 @@ with ApiClient(...) as api_client:
     # process the response ...
 ```
 
-Operation parameters can be passed as positional or keyword arguments. There are also several
-extra arguments which change invocation logic:
+Operation parameters can be passed as positional or keyword arguments. API methods provide
+extra common arguments which control invocation logic:
 
-- `_parse_response` - Allows to enable and disable response data parsing. When enabled,
+- `_parse_response` (`bool`) - Allows to enable and disable response data parsing. When enabled,
   the response data is parsed into a model or a basic type and returned as the first value.
   When disabled, the response is not parsed, and `None` is returned. Can be useful,
   for instance, if you need to parse data manually, or if you expect an error in the response.
-- `_check_status` - Allows to enable or disable response status checks. When enabled, the
+  Default is `True`.
+- `_check_status` (`bool`) - Allows to enable or disable response status checks. When enabled, the
   response status code is checked to be positive as defined in the [HTTP standards](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes).
-  In case of negative status, an exception is raised.
-- `_request_timeout` - Allows to control timeout
-- `_content_type` - Allows to specify the `Content-Type` header value for the request. Endpoints
-  can support different content types and behave differently depending on the value. For file
-  uploads `_content_type="multipart/form-data"` must be specified.
+  In the case of negative status, an exception is raised. Default is `True`.
+- `_validate_inputs` (`bool`): specifies if type checking should be done one the data
+  sent to the server. Default is `True`.
+- `_validate_outputs` (`bool`): specifies if type checking should be done one the data
+  received from the server. Default is `True`.
+- `_request_timeout` (`None | int | float | Tuple[int | float, int | float]`) -
+  Allows to control timeout. If one number provided, it will be total request timeout. It can also
+  be a tuple with (connection, read) timeouts. Default is `None`, which means no timeout.
+- `_content_type` (`None | str`) - Allows to specify the `Content-Type` header value
+  for the request. Endpoints can support different content types and behave differently
+  depending on the value. For file uploads `_content_type="multipart/form-data"` must be specified.
+  Read more about file uploads [here](#sending-data). Default `None`, which is to try to detect
+  automatically.
```

Comment:
Does it really do autodetection? I thought the default was just `application/json`.

Reply:
This is just copied from the API description, I need to double-check.

Reply:
Kind of, it can take the first applicable value from what the schema provides, or to try to convert into the `multipart` if it is requested by headers or by the method parameter, otherwise `application/json` is used. I think, we can safely assume it is `application/json` by default in our cases.